bokehjs/src/coffee/polyfill.ts

```diff
@@ -0,0 +1,9 @@
+declare function require(moduleName: string): any
+
+if (typeof WeakMap !== "function") {
+  require("es6-weak-map/implement")
+}
+
+if (typeof Set !== "function") {
+  require("es6-set/implement")
+}
```

Comment:
OK, I expected that would be the case, and that's good to hear,  I just wanted to make sure the requirement was voiced explicitly.